apps/ideas/src/components/modules/IdeaspaceNavCard/IdeaspaceNavCard.css

```diff
@@ -0,0 +1,10 @@
+.idea-nav-card {
```

Comment:
Could we add this as a style to the `StyledWelcomePage` file? I know I helped contribute to creating this file, but using a .css file is not aligned with how we are doing things elsewhere in the project.